bot.py

```diff
@@ -112,13 +124,119 @@ def on_message(ws, message):
         if admin_message(response):
             return
 
-    elif "team_join" == response["type"]:
-        if NEWUSERGREETING:
-            new_user_message(response)
-        USER_LIST = sc.api_call(  # noqa: 841
-            "users.list",
-        )
 
+    #TODO: Purge all this shit
```

Comment:
This doesn't need to be in our code base, also are we getting rid of the user greeting? how does this relate to analytics?